src/stories/components/delegate-summary/delegate-summary.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import styled from '@emotion/styled';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { LinkTypeEnum } from '@ses/core/enums/link-type.enum';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import { CircleAvatar } from '../circle-avatar/circle-avatar';
+import { CuTableColumnLinks } from '../cu-table-column-links/cu-table-column-links';
+import { CustomLink } from '../custom-link/custom-link';
+
+interface Props {
+  code?: string;
+}
+
+const DelegateSummary: React.FC<Props> = ({ code = 'del' }) => {
+  const isUp1440 = useMediaQuery(lightTheme.breakpoints.between('desktop_1440', 'desktop_1920'));
+  const links = [
+    {
+      linkType: LinkTypeEnum.WWW,
+      href: 'sm.website',
+    },
+    {
+      linkType: LinkTypeEnum.Forum,
+      href: 'sm.website',
+    },
+    {
+      linkType: LinkTypeEnum.Discord,
+      href: 'sm.website',
+    },
+    {
+      linkType: LinkTypeEnum.Youtube,
+      href: 'sm.website',
+    },
+  ];
+
+  return (
+    <Container>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+        }}
+      >
+        <ContainerLogoDescription>
+          <CircleContainer>
+            <CircleAvatar
+              style={{
+                filter: 'filter: drop-shadow(2px 4px 7px rgba(26, 171, 155, 0.25));',
+              }}
+              width={isUp1440 ? '68px' : '32px'}
+              height={isUp1440 ? '68px' : '32px'}
```

Comment:
can this be done using css media queries instead? Maybe adding the styles to `CircleContainer` if `CircleAvatar` can't be extended with `styled`. Sometimes `useMediaQuery`, as it is done with pure javascript, have a delay to update the variable and to have effect as it need to trigger a React re-render